server/seeders/20190407220433-articles.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
       slug: 'Gildard is working on it-12345678',
       body: 'Dickson is a boy',
       description: 'gildard@dickson.com',
+      readingTime: '1 min read',
```

Comment:
@geekdanitek Kindly confirm if the `reading-time` package returns this exactly as the reading time. If it would be possible to return a `TIME` data type, I think that would be easier to test for and also would make the presentation on the frontend more flexible.

Reply:
Yes, the `reading-time` package returns in this format `1 min read` and i would like more clarity on this 

> would be possible to return a TIME data type, I think that would be easier to test for and also would make the presentation on the frontend more flexible.

Reply:
Done